Library/Formula/nesc.rb

```diff
@@ -1,8 +1,13 @@
 class Nesc < Formula
   desc "Programming language for deeply networked systems"
-  homepage "http://nescc.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/nescc/nescc/v1.3.5/nesc-1.3.5.tar.gz"
-  sha256 "c22be276d565681a2b84ddbf2a037256d24ecbf0da35e30157589609eec63096"
+  homepage "https://github.com/tinyos/nesc"
+  url "https://github.com/tinyos/nesc/archive/v1.3.6.tar.gz"
+  sha256 "80a979aacda950c227542f2ddd0604c28f66fe31223c608b4f717e5f08fb0cbf"
+
+  depends_on :automake => :build
+  depends_on :autoconf => :build
+  depends_on :java     => :build
```

Comment:
Don't worry about the equal spacing on the `=>` here. It doesn't do much for style and some text editors complain about it.